lib/runtime.cpp

```diff
@@ -42,13 +42,21 @@ RuntimeSettings::RuntimeSettings(const po::variables_map& vm) : main_dir(vm["mai
         config_file = main_dir / "conf/config.json";
     }
 
+    if (vm.count("user_conf")) {
+        user_config_file = vm["user_conf"].as<std::string>();
+    } else {
+        user_config_file = config_file.parent_path() / "user-config" / config_file.filename();
```

Comment:
Furthermore, if no user-config is supplied via command-line, it might still happen, that one gets loaded ...
This is some kind of default behavior, but it should be explicitly stated with log info

Reply:
The problem with stating this with a log info message here is that it will be printed for every module

Reply:
Right, I need to rebase my refactor module branch, there only the manager reads the config.  This is really brittle here, because different module might have a different understanding about what the real config is

Reply:
That would be even better. If we could integrate a mechanism for providing a user config via the command line there I'll happily drop this PR in favor of a more robust solution.
Maybe we could even integrate a way of providing multiple user-configs (via the command line) that get applied on top of the main config in the specified order. This would make the config handling much more powerful.